setup.py

```diff
@@ -74,10 +74,11 @@
         'tox',
         'wheel==0.30.0',
         'azure-storage-blob>=1.3.1,<2.0.0',
+        'isort=4.3.21'
     ],
     extras_require={
         ":python_version<'3.0'": ['pylint==1.9.2', 'futures'],
-        ":python_version>='3.0'": ['pylint==2.5.3']
+        ":python_version>='3.0'": ['pylint==2.3.0']
```

Comment:
Do we need to release new azdev since the released version is also broken?

Reply:
Luckily we currently use the `main` branch of `azdev` in Azure CLI CI so the fix will applied. But yes, we do need to release a new version of `azdev`.

Reply:
This version pin issue may exist in other third-party packages, we should look through and pin all of them.